src/data.jl

```diff
@@ -504,7 +504,7 @@ function ProjectedDensityOfStates(
     # TODO: perhaps we can improve this with a trapezoidal approximation?
     int = hcat(vec(sum(M[:,1:n], dims=2)) for n in 1:size(M,2))
```

Comment:
Considering that M is a matrix and the original function takes `dos::AbstractMatrix{<:Real}` as input, perhaps `M` was supposed to be `dos`?

Either way, I don't know if we need this constructor now so disabling it should probably be OK.